worlds/dlcquest/Locations.py

```diff
@@ -76,3 +76,14 @@ class DLCQuestLocation(Location):
 for i in range(1, 890):
     item_coin_freemium = f"Live Freemium or Die: {i} Coin"
     location_table[item_coin_freemium] = offset + 825 + 58 + i
+
+
+offset_special = 3829200000
+
+for i in range(1, 8251):
+    item_coin_piece = f"DLC Quest: {i / 10} Coin Piece"
```

Comment:
If they are called "X Coin Piece", X should be a whole number from 1 to N * 10, not a decimal number from 0.1 to N